packages/at_client/lib/src/client/at_client_impl.dart

```diff
@@ -472,9 +472,13 @@ class AtClientImpl implements AtClient, AtSignChangeListener {
     if (atKey.metadata!.namespaceAware) {
       atKey.namespace ??= preference?.namespace;
     }
-    if (atKey.key!.contains(r'^[A-Z]') || atKey.namespace!.contains(r'^[A-Z]') ) {
+
```

Comment:
This works :-)

Suggest making upperCaseRegex a static const

Reply:
sure. let me make that change too

Reply:
since it's not possible to use a static var in an instance method, I've made upperCaseRegex a class-level static variable in the latest commit.